rllib/evaluation/worker_set.py

```diff
@@ -413,7 +413,8 @@ def sync_weights(
         # Only sync if we have remote workers or `from_worker` is provided.
         weights = None
         if self.num_remote_workers() or from_worker is not None:
-            weights = (from_worker or self.local_worker()).get_weights(policies)
+            worker_or_trainer = from_worker or self.local_worker()
```

Comment:
Dumb question: `worker_or_trainer` is always a worker, no? Or can `from_worker` be a RLTrainer as well? If so, we should change the name of that arg to `from_worker_or_trainer`. Hope this makes sense.

Reply:
yeah. Anything that implements get_weights() can become from_worker. Updated the names to make it more consistent.